.github/workflows/e2e_pr.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         id: yarn-cache-dir-path
         run: echo "::set-output name=dir::$(yarn cache dir)"
       - name: Set url
-        run: echo "CYPRESS_BASE_URL=https://deploy-preview-${{github.event.pull_request.number}}--wazimap-staging.netlify.app" >> $GITHUB_ENV
+        run: echo "CYPRESS_BASE_URL=https://deploy-preview-${{github.event.pull_request.number}}--wazimap-production.netlify.app" >> $GITHUB_ENV
```

Comment:
sorry, unrelated change - getting rid of the wazimap-staging app because I think Mila didn't realise you could do branch deploys